tests/Resources/ti.ui.shortcutitem.test.js

```diff
@@ -13,12 +13,12 @@ const should = require('./utilities/assertions');
 
 describe('Titanium.UI.ShortcutItem', () => {
 
-	describe.android('createShortcutItem', () => {
+	describe('createShortcutItem', () => {
+    
```

Comment:
<!--
  1 failure:  tests/Resources/t...
  0 warning: 
  
  
  DangerID: danger-id-Danger;
  File: tests/Resources/ti.ui.shortcutitem.test.js;
  Line: 17;
-->
- :no_entry_sign: tests/Resources/ti.ui.shortcutitem.test.js line 17 – Trailing spaces not allowed. (no-trailing-spaces)